test/remote/gateways/remote_quickbooks_test.rb

```diff
@@ -141,6 +141,24 @@ def test_successful_purchase_with_expired_token
     assert_success response
   end
 
+  def test_successful_purchase_without_state_address
+    options = {
+      order_id: '1',
+      billing_address:
+        {
+          zip: 90210,
+          # the test will pass if these values are nil
+          # but they fail when passed as empty strings
```

Comment:
do we want to keep this comment in? I think now that this change fixes this problem it could be omitted (or changed to something like `...previously it would fail if passed as empty strings`

Reply:
My intent was to leave a comment in there to describe that it's the empty string version that fails, but you're right that I need to update the text. Thanks for catching that!